### PR TITLE
Intellij blog post

### PR DESCRIPTION
**Pull request by andricDu**

### CODE REVIEW

**Review by Fgerthoffert — APPROVED** [review]

**Comment by Fgerthoffert** [comment]
Great article, don't forget to update the date before merging.

File: _posts/dusan_andric/3001-01-01-intellij.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    version: small
+    title: Software Engineering Blog
+    image: header-logo-crop.png
+    icon: icon-blog
+---
+
+## Introduction
+The fields of Cancer Research and Human Genomics, while presenting some very hard and interesting problems that touch the domains of Computer Science and Software Engineering, do not always have the resources to aquire or use the latest and greatest software tools. In particular it can be really difficult to justify such expenditures. 
+
+However, the kind folks at JetBrains were nice enough to support our development in these important fields, so as a thank you, we want to talk about what gets us so excited about using their tools. Choosing the right tools can significantly improve developer quality of life as well as productivity and we want to talk about the JetBrains tools that we love that do just that.
```
Comment (cheapsteak) [inline _posts/dusan_andric/3001-01-01-intellij.md]: This paragraph could maybe be workshopped a bit

> However, the /kind folks/ at JetBrains were /nice enough/ to support our development in these important fields, so as a thank you, we want to talk about what gets us so excited about using their [tools]. Choosing the right [tools] can significantly improve developer quality of life as well as productivity and we want to talk about the JetBrains [tools] (that) we love (that) do just (that).

emphasized the parts that kind of stuck out to me

File: _posts/dusan_andric/3001-01-01-intellij.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+---
+
+## Introduction
+The fields of Cancer Research and Human Genomics, while presenting some very hard and interesting problems that touch the domains of Computer Science and Software Engineering, do not always have the resources to aquire or use the latest and greatest software tools. In particular it can be really difficult to justify such expenditures. 
+
+However, the kind folks at JetBrains were nice enough to support our development in these important fields, so as a thank you, we want to talk about what gets us so excited about using their tools. Choosing the right tools can significantly improve developer quality of life as well as productivity and we want to talk about the JetBrains tools that we love that do just that. 
+
+## IntelliJ IDEA
+
+IntelliJ is a very intuitive, robust, and dare I say it, enjoyable way of programming in Java. Both the community and Ultimate version include many features that accelerate the process of developing our software. While much of the appeal of IntelliJ is, as the cliche goes, that it just works, we highlight a few life-enhancing features that our team takes advantage of or is aided by.
```
Comment (cheapsteak) [inline _posts/dusan_andric/3001-01-01-intellij.md]: > Both the community

"Community" should be uppercased

File: _posts/dusan_andric/3001-01-01-intellij.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+IntelliJ is a very intuitive, robust, and dare I say it, enjoyable way of programming in Java. Both the community and Ultimate version include many features that accelerate the process of developing our software. While much of the appeal of IntelliJ is, as the cliche goes, that it just works, we highlight a few life-enhancing features that our team takes advantage of or is aided by. 
+
+### Duplicate Detection 
+
+IntelliJ can detect copy-and-paste code that someone lazily (cough) added into the code-base and allows you to quickly navigate between duplicates. 
+
+<figure>
+    <img src="{{site.urlimg}}dusan_andric/jetbrains/duplicates.png" />
+    <figcaption>duplicate code detection</figcaption>
+</figure>
```
Comment (cheapsteak) [inline _posts/dusan_andric/3001-01-01-intellij.md]: Very cool!

File: _posts/dusan_andric/3001-01-01-intellij.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+<figure>
+    <img src="{{site.urlimg}}dusan_andric/jetbrains/detection.png" />
+    <figcaption>context-sensitive plugin suggestion</figcaption>
+</figure> 
+
+It's a small touch, but appreciated. 
+
+### Code-style Importing 
+
+So you like IntelliJ but your team likes Eclipse. Or vice versa. Keep things in sync and reduce un-necessary or distracting formatting challenges by taking advantage of IntelliJ's ability to import code formatting files from Eclipse, from CheckStyle, or from other IntelliJ installs.
```
Comment (cheapsteak) [inline _posts/dusan_andric/3001-01-01-intellij.md]: > from Eclipse, from CheckStyle, or from other IntelliJ installs. 
 
recommended to change to : `from Eclipse, CheckStyle, or other IntelliJ installations.`

DRY :D

File: _posts/dusan_andric/3001-01-01-intellij.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+The UI, in addition to showing your dependencies in a concise way, also displays possible updates and allows you to search for additional packages found in PyPI or any other repository you may have configured. 
+
+### PEP8 and Style
+
+PyCharm will continuously monitor your code style to make sure it adheres to the PEP8 style guide for Python. You can of course configure additional style rules and enforments, such as when to line break around parentheses and braces for example.
+
+
+### Frameworks
+
+The framework support provided by PyCharm, in particular web frameworks, is simply best in class. If you are looking creating a Django webapp serving Jinja2 templates or a small Flask app with a React SPA for a front-end, PyCharm is your one stop shop for everything. Need to access a MySQL instance or manage a docker deployment? PyCharm will still have you covered.
```
Comment (cheapsteak) [inline _posts/dusan_andric/3001-01-01-intellij.md]: >If you are looking creating a Django webapp

If you are looking at／contemplating？

> PyCharm will still have you covered

recommend: `PyCharm has you covered`

**Pull request merged**